demos/smartlab_demo/python/README.md

```diff
@@ -40,7 +42,7 @@ omz_downloader --list models.lst
 * smartlab-object-detection-0003
```

Comment:
Don't forget to replace `https://storage.openvinotoolkit.org/data/test_data/videos/smartlab/` with `https://storage.openvinotoolkit.org/data/test_data/videos/smartlab/v3` on top